packages/keycloak-user-management/src/components/UserList/tests/index.test.tsx

```diff
@@ -358,4 +358,52 @@ describe('components/UserList', () => {
     );
     expect(mockNotificationError).toHaveBeenCalledWith(lang.ERROR_OCCURED);
   });
+
+  it('sorting works', async () => {
```

Comment:
do we really need to have this test ? i.e shouldn't something like this be tested in table layout 🤔💭 unless there is some coverage issue

Reply:
@ashfahan I have no strong opinion about it. More tests are better than less tests I presume?

Reply:
It seems the behaviour under test more applies to the UserList component (ie. how `Userlist` uses the tableLayout component) as opposed to how the table layout itself works.  Also if we view this from an integration test standpoint then am ok with the added test, of course the Table layout should be already well covered by unit tests too.